core/src/main/resources/jenkins/model/Messages_fr.properties

```
@@ -52,3 +66,20 @@ ParameterizedJobMixIn.build_now=Lancer un build
 BlockedBecauseOfBuildInProgress.shortDescription=Le build #{0} est déjà en cours {1}
 BlockedBecauseOfBuildInProgress.ETA=\ (fin prévue à : {0})
 BuildDiscarderProperty.displayName=Supprimer les anciens builds
+
+EnforceSlaveAgentPortAdministrativeMonitor.displayName=Impose le port de l''agent TCP
+CLI.disable-job.shortDescription=Désactive un job.
+CLI.enable-job.shortDescription=Active un job.
+
+GlobalCloudConfiguration.DisplayName=Clouds
+
+BuiltInNodeMigration.DisplayName=Migration du nom et du libellé du nœud contrôleur
+
+SimpleGlobalBuildDiscarderStrategy.displayName=Défausseur de build spécifique
+JobGlobalBuildDiscarderStrategy.displayName=Défausseur de build de projet
+
+Hudson.ReadPermission.Description=\
+  Le droit en lecture est nécessaire pour voir la plupart des pages de Jenkins. \
+  Ce droit est utile quand vous ne voulez pas que les utilisateurs non authentifiés puissent voir les pages Jenkins \
+  — retirez ce droit à l''utilisateur anonymous, puis \
```

Any reason for not using a standard dash? (nit)
```suggestion
  - retirez ce droit à l''utilisateur anonymous, puis \
```

---

https://github.com/jenkinsci/jenkins/pull/8577/commits/9798d9a507b945d84a63aba26fa1ecb7d757d3ad